tests/unit/ViolationUtilsTest.js

```diff
@@ -0,0 +1,505 @@
+import Onyx from 'react-native-onyx';
+import ViolationsUtils from '@libs/Violations/ViolationsUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+describe('getViolationsOnyxData', () => {
+    let transaction;
+    let transactionViolations;
+    let policyRequiresTags;
+    let policyTags;
+    let policyRequiresCategories;
+    let policyCategories;
+
+    beforeEach(() => {
+        transaction = {category: 'Food', tag: 'Lunch', transactionID: '123'};
+        transactionViolations = [
+            {name: 'duplicatedTransaction', type: 'violation', userMessage: ''},
+            {name: 'receiptRequired', type: 'violation', userMessage: ''},
+        ];
+        policyRequiresTags = false;
+        policyTags = {};
+        policyRequiresCategories = false;
+        policyCategories = {Food: {enabled: true}};
+    });
+
+    const execute = () => ViolationsUtils.getViolationsOnyxData(transaction, transactionViolations, policyRequiresTags, policyTags, policyRequiresCategories, policyCategories);
```

Comment:
I'm not super passionate about this, but can we remove this execute function? We're testing ViolationsUtils.getViolationsOnyxData so I think it's fine, and also preferable, to have those calls in the tests instead of `execute`